internal/connectors/plugins/public/bankingcircle/balances.go

```diff
@@ -31,20 +33,20 @@ func (p Plugin) fetchNextBalances(ctx context.Context, req models.FetchNextBalan
 		// circle response. We will use the current time instead.
 		// lastTransactionTimestamp, err := time.Parse("2006-01-02T15:04:05.999999999+00:00", balance.LastTransactionTimestamp)
 		// if err != nil {
-		// 	return models.FetchNextBalancesResponse{}, fmt.Errorf("failed to parse opening date: %w", err)
+		// 	return models.FetchNextBalancesResponse{}, models.NewPluginError(fmt.Errorf("failed to parse opening date: %w", err))
```

Comment:
Is all those comment still needed ?

Reply:
Ah good question. Was this intended to be commented out or do we need to override the timestamp @paul-nicolas ?

Reply:
This is why we do lastTransactionTimestamp := time.Now().UTC() instead of parsing the balance transaction timestamp. But yeah we can keep only the comment and not the commented code I think

Reply:
removed